src/game-data/effects/cards/1-2-141.ts

```
@@ -5,29 +5,47 @@ import { Effect } from '../engine/effect';
+          {
+            title: '封札の煉獄炎',
+            description: '3000ダメージ',
+            effect: () => Effect.damage(stack, stack.processing, target, 3000),
+            condition: opponent.field.some(unit => unit.id === target.id),
+          },
+          {
+            title: '封札の煉獄炎',
+            description: 'デッキから1枚トリガーゾーンにセット',
+            effect: () =>
+              EffectHelper.random(owner.deck).forEach(card =>
+                Effect.move(stack, stack.processing, card, 'trigger')
```

Biome (2.3.13)

[error] This callback passed to forEach() iterable method should not return a value.

Either remove this return or remove the returned value.


(lint/suspicious/useIterableCallbackReturn)